.vsts-dotnet.yml

```diff
@@ -69,9 +84,9 @@ stages:
 
     - task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@1
       inputs:
-        signType: $(SignType)
+        signType: ${{parameters.SignType}}
```

Comment:
I think we can just pass this directly into the `SignType` parameter, similar to `officialSkipApplyOptimizationData`

Reply:
Oh wait, this is an input to the task.

Reply:
Okay, answered my own rodeo of questions. `$(SignType)` is only working for the build script because it's still defined as a pipeline variable. I suspect if you change the `SignType` pipeline **parameter**, it would still be `Real` if the pipeline **variable** was unchanged

Reply:
Ah, I thought I'd changed it there too. Let me do so now.